ddtrace/constants.py

```diff
@@ -39,27 +38,7 @@
 USER_KEEP = 2
 
 
-_DEPRECATED = {
-    # TODO: Removal of this attribute does not require the addition of a
-    # constant as it has only a single use for passing trace filters in the
-    # settings to the tracer.
-    "FILTERS_KEY": "FILTERS",
-    # TODO: Moved to other modules but cannot reference here due to circular imports
-    # ddtrace.contrib.logging.patch._LOG_SPAN_KEY
-    # ddtrace.span._NUMERIC_TAGS
-    "NUMERIC_TAGS": (ANALYTICS_SAMPLE_RATE_KEY,),
-    "LOG_SPAN_KEY": "__datadog_log_span",
-}
-
-
 def __getattr__(name):
```

Comment:
Should we remove this function or at least deprecate it?

Reply:
yeah, I think you can get rid of this and `ensure_pep562` ?

we might need to look for `ensure_pep562` elsewhere too